types/index.d.ts

```diff
@@ -18,6 +18,7 @@ import {
   Styles,
   Toolbar,
   Tooltip,
+  I18n,
```

Comment:
Other types also should be exported from index.d.ts to be accessible by import from `@editorjs/editorjs` package

Reply:
done